src/io/writev.rs

```diff
@@ -32,22 +33,24 @@ impl<T: IoBuf> Op<Writev<T>> {
             })
             .collect();
 
-        Op::submit_with(
-            Writev {
-                fd: fd.clone(),
-                bufs,
-                iovs,
-            },
-            |write| {
-                opcode::Writev::new(
-                    types::Fd(fd.raw_fd()),
-                    write.iovs.as_ptr(),
-                    write.iovs.len() as u32,
-                )
-                .offset(offset as _)
-                .build()
-            },
-        )
+        CONTEXT.with(|x| {
+            x.handle().expect("Not in a runtime context").submit_op(
```

Comment:
This looks good too. Makes it clear the Op can only be made if there is a local runtime context and the strong handle is available.